apps/console/src/features/console-settings/components/console-administrators/invited-administrators/invited-administrators-table.tsx

```diff
@@ -155,6 +155,9 @@ const InvitedAdministratorsTable: React.FunctionComponent<InvitedAdministratorsT
     const [ deletingUser, setDeletingUser ] = useState<UserInviteInterface>(undefined);
     const [ alert, setAlert, alertComponent ] = useConfirmationModalAlert();
 
+    const featureConfig: FeatureAccessConfigInterface = useSelector((state: AppState) => {
```

Comment:
If this only contains the configs for the `users` feature, let's use a more specific variable name instead of `featureConfig`

Reply:
Since we have lots of places using this pattern, we need to test the flows again after changing this. Since this PR is somewhat urgent for QA testing. Will send a follow up PR to fix this.